app/build.gradle

```diff
@@ -42,6 +44,7 @@ android {
         }
     }
     testOptions {
+        //execution 'ANDROIDX_TEST_ORCHESTRATOR'
```

Comment:
I kept it as this could help us with testing permissions if android studio 3.6.2 came out before the end of the semester but as the tests pass this way I'll remove it